core-blocks/columns/style.scss

```diff
@@ -1,12 +1,9 @@
 .wp-block-columns {
-
-	.editor-inner-blocks {
-		display: grid;
-		grid-auto-flow: dense;
-	}
+	display: grid;
+	grid-auto-flow: dense;
 
 	@for $i from 2 through 6 {
-		&.has-#{ $i }-columns .editor-inner-blocks {
+		&.has-#{ $i }-columns {
```

Comment:
This is a base stylesheet, meaning that in the editor, we have styles for both `.has-x-columns` and `.has-x-columns .editor-inner-blocks` to apply the `grid-auto-columns`. Will there be issues from this?

Reply:
I don't think because the style in the editor will be applied to .wp-block-columns  and we set it to display block so my expectation is that grid styles are simply ignored.